res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1830 @@
+						<en>The picture hangs crooked</en>
+					</title>
+					<description>
+						<de>Ein Freund des Hauses kommt zu Besuch. Das Dienstmädchen lässt ihn ein und sagt, er soll im Salon warten. Als er dort ein schiefes Bild gerade rücken will, zerstört er die Einrichtung.</de>
+						<en>A friend of the house comes to visit. The maid lets him in and tells him to wait in the parlor. When he tries to straighten a crooked picture there, he destroys the furnishings.</en>
+					</description>
+					<ratings critics="22" speed="25" />
+				</programme>
+				<programme id="cujo-serien-best-torilo-04" product="2" licence_type="2" tmdb_id="0" imdb_id="" creator="8827" created_by="Cujo">
+					<title>
+						<de>Das Reiskorn</de>
+						<en>The grain of rice</en>
+					</title>
+					<description>
+						<de>Bei einem Rendezvous entdeckt Hannelore ein Reiskorn am Mund ihres Verehrers. Trotz mehreren Versuchen es zu entfernen, taucht es immer wieder auf.</de>
+						<en>On a date, Hannelore discovers a grain of rice on her beau's mouth. Despite several attempts to remove it, it keeps reappearing.</en>
+					</description>
+					<ratings critics="22" speed="25" />
+				</programme>
+				<programme id="cujo-serien-best-torilo-05" product="2" licence_type="2" tmdb_id="0" imdb_id="" creator="8827" created_by="Cujo">
+					<title>
+						<de>Das Gummientchen</de>
+						<en>The rubber ducky</en>
+					</title>
+					<description>
+						<de>Meier-Schüdenleidt will in seinem Hotelzimmer baden. In seiner Badewanne sitzt aber bereits Doktor Blögner mit seinem Gummientchen.</de>
+						<en>Meier-Schüdenleidt wants to take a bath in his hotel room. But Doctor Blögner is already sitting in his bathtub with his rubber ducky.</en>
+					</description>
+					<ratings critics="22" speed="25" />
+				</programme>
+			</children>
+		</programme>		
+		<programme id="cujo-serien-tigerzahn" product="2" licence_type="3" tmdb_id="0" imdb_id="tt0135097" creator="8827" created_by="Cujo">
+			<title>
+				<de>Tigerzahn</de>
+				<en>Tiger Tooth</en>
+			</title>
+			<title_original>
+				<de>Löwenzahn</de>
+			</title_original>
+			<description>
+				<de>Paul Witzig wohnt in dem kleinen Städtchen Löwendorf. Er trägt eine Nickelbrille und eine Latzhose. Paul beobachtet interessiert die Natur und Umwelt rund um seinen Bauwagen.</de>
+				<en>Paul Witzig lives in the small town of Löwendorf. He wears nickel glasses and dungarees. Paul observes nature and the environment around his construction trailer with interest.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">cujo-person-karl-lora</member>
+				<member index="1" function="2">cujo-person-paul-witzig</member>
+				<member index="2" function="2">cujo-person-horst-krause</member>
+			</staff>
+			<groups target_groups="1" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="D" year="1981" distribution="2" maingenre="9" subgenre="" flags="264" blocks="1" price_mod="0.71" />

```

Comment:
Ich tue mich immer etwas schwer mit der Kult-Flag-Inflation. Zwar sind viele der Filme in der Datenbank Kult, aber das bedeutet ja nicht, dass sie in TVTower selbst alle Kult sein müssen.